content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,42 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+Palette allows the pause and resume of Palette Virtual Clusters when not in use. This feature enables the users to optimize resource utilization by pausing the Virtual Clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette Virtual Clusters. 
+
+<br />
+
+### System and Resource Impact
+
+* The quota allocation is independent of a Virtual Cluster's Pause or Resume status.
```

Comment:
All instances of "Virtual Cluster" should be changed to the virtual cluster. They are all title cased when they should be lower case. We are not using Virtual Clusters as a term after the initial definition of Palette Virtual Cluster. After the initial usage, we can simply refer to the usage as virtual clusters.